PytHW5/Task1.py

```diff
@@ -0,0 +1,10 @@
+def step(A, B):
+    if B == 1:
+        return 1
+    else:
+        return A*step(A, B-1)
```

Comment:
Конструкция `else` не нужна, так как `return` завершает работу функции
```suggestion
    return A * step(A, B - 1)
```